### ISSUE TITLE
[question] Will pull requests get merged and status of tye

### ISSUE THREAD

**Issue by ikkentim** [dotnet/tye#1186]
Hi,

I've been using Tye for the past year and really like it. I've been using it in a few projects and am interested in seeing this project succeeding in its goals. There are a few things that cause me concern however, and I'd like to know what the maintainers' plans are for Tye.

One thing that causes me concern is that I've been trying to getting simple improvements merged for over 8 months now, first in #916 and then in #1153, but sadly no maintainer has taken the time to review the changes. There are also a number of other pull requests that contain useful features or improvements which would make Tye even better.

I've also noticed that the (what I believe to be the) original maintainers seem to have left the project behind ( 072de978b7d2882c3433e37e5f1c8349b622bb98 ) and that since ~0.7 no changes were made to Tye other than a few improvements in the APIs used by the VS code extension.

My questions are:
- Is this project actively maintained?
- Will this project still be around in a year or two?

As far as I am aware this is the best tool for developing and running a microservice application locally. I am grateful for it's existence and the time people have put into it and I would really like to keep using Tye.

**Comment by jooooel** [comment]
I agree. I've been using Tye a lot the past year and I would really like it (or something similar to exist). 

For me personally, I don't care much about the different integrations (Kubernetes, dapr, automatic service discovery etc). But I really appreciate `tye run`. Based on a `tye.yaml` where our applications are defined, I can mix and match and choose what I want to run (Docker containers, dotnet processes etc) simply by specifying different tags. 

The one thing that I really miss and was hoping for, is the possibility to start/stop/restart individual services. Where Tye runs more like a deamon in the background, keeping track of it all. 

I don't know that there are other tools available that lets you do this easily? Any advice would be appreciated if they exist. I have been thinking about forking Tye, keep the things that I like and add the things that I miss. But as always, my amount of spare time works against me 😊 

Hoping for some kind of announcement (positive or negative)!

**Comment by BigMorty** [comment]
@ikkentim, looks like David just approved your outstanding PR, which I think supersedes the other one that you closed. Thanks for your contributions!

@jooooel, we are happy to hear you find Tye useful and we are working hard to get some of the top issues resolved. Would you please open a separate issue for your idea of being able to start/stop/restart individual services? That way we can see if others have the same need and prioritize it appropriately. Also, have you tried the --watch flag which will restart services that have had changes made to them?

**Comment by jooooel** [comment]
@BigMorty Thanks for getting back to us, sounds like Tye hasn't been totally buried then 😉 

I've actually opened an [issue regarding start/stop](https://github.com/dotnet/tye/issues/876) already, and I did a little PoC just to get a feel for how Tye works and what it would take. My solution works, but I think it would be better to go towards more of a "deamon" approach as I mentioned. 

I've tried to use `--watch`, but due to the nature of the codebase I use, it doesn't really perform well when doing changes in common libs (lot's of things restart and hangs VS for me).
